### PR TITLE
docs: Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Esemesek**
Add missing `CONTRIBUTING.md` file.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+1. Fork the repo and create your branch from `master` (a guide on [how to fork a repository](https://help.github.com/articles/fork-a-repo/)).
+2. Run `yarn` or `npm install` to install all required dependencies.
+3. Now you are ready to do the changes.
+
+## Testing your changes
+
+You can test your changes by installing `example` app on a simulator or device:
+
+Installing on a ios:
```
Comment (thymikee) [inline CONTRIBUTING.md]:
```suggestion
Installing on a iOS:
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+- Open `example/ios/example.xcodeproj`
+- Click `run` in the top left corner of `Xcode`.
+
+
```
Comment (thymikee) [inline CONTRIBUTING.md]: run prettier

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+3. Now you are ready to do the changes.
+
+## Testing your changes
+
+You can test your changes by installing `example` app on a simulator or device:
+
+Installing on a iOS:
+
+- Open `example/ios/example.xcodeproj`
+- Click `run` in the top left corner of `Xcode`.
```
Comment (thymikee) [inline CONTRIBUTING.md]: or use `yarn react-native run-ios`?

Comment (Esemesek) [reply CONTRIBUTING.md]: It wasn't quite working for me.

Comment (thymikee) [reply CONTRIBUTING.md]: why :< would be nice to figure it out and fix

Comment (Esemesek) [reply CONTRIBUTING.md]: True, I'll create an issue for that

**Pull request merged**